src/global/types/actions.ts

```diff
@@ -2290,6 +2294,7 @@ export interface ActionPayloads {
     url: string;
     shouldSkipModal?: boolean;
     ignoreDeepLinks?: boolean;
+    linkContext?: LinkContext;
```

Comment:
Add this parameter for `openUrl` calls in `clickBotInlineButton` and `answerCallbackButton`